compose/cli/main.py

```diff
@@ -285,6 +286,7 @@ def build(self, options):
             build_args=build_args,
             gzip=options.get('--compress', False),
             parallel_build=options.get('--parallel', False),
+            quiet=options.get('--quiet', False)
```

Comment:
please use `silent` as the parameter name internally to match what we did with `pull`